common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -570,6 +569,20 @@
                                 gettext('Show posts by {username}.'), {username: username}
                             );
                             self.addSearchAlert(message, 'search-by-user');
+                            DiscussionUtil.safeAjax({
+                                url: DiscussionUtil.urlFor('followed_threads', response.users[0].id),
```

Comment:
@noraiz-anwar I have to add little bit code here if the user found, I have retrieved all the followed thread.